src/vorta/store/models.py

```diff
@@ -26,11 +27,11 @@ class JSONField(pw.TextField):
     From: https://gist.github.com/rosscdh/f4f26758b0228f475b132c688f15af2b
     """
 
-    def db_value(self, value):
+    def db_value(self, value) -> (None or str):
```

Comment:
Is that a valid type hint?

Reply:
I don’t think so. It should be `None | str`

Reply:
And for that one needs to add `from __future__ import annotions` afaik.

Reply:
Only for Python < 3.10

Reply:
...which we support

Reply:
Hmm.. I think `Union[str, None]` would be better

Reply:
Even better: `Optional[str]`